agent/charms/testflinger-agent-host-charm/src/charm.py

```diff
@@ -30,29 +30,29 @@ class TestflingerAgentHostCharm(CharmBase):
 
     def __init__(self, *args):
         super().__init__(*args)
-        self.framework.observe(self.on.install, self._on_install)
-        self.framework.observe(self.on.start, self._on_start)
-        self.framework.observe(self.on.config_changed, self._on_config_changed)
+        self.framework.observe(self.on.install, self.on_install)
+        self.framework.observe(self.on.start, self.on_start)
+        self.framework.observe(self.on.config_changed, self.on_config_changed)
```

Comment:
Just out of curiosity, is there a particular reason for avoiding methods that start with `_`

Reply:
mostly just opinion that I hate reading it  like that and it's kind of pointless since this isn't a module that other things are importing where we're trying to give hints about what should be called publicly vs what should be only used internally. I'm not stuck on that opinion, just what I was thinking when I started looking at it again.